cmd/tempo-serverless/cloud-run/go.mod

```diff
@@ -0,0 +1,147 @@
+module github.com/grafana/tempo/cmd/tempo-serverless/cloud-run
+
+go 1.18
```

Comment:
Time to party.

Reply:
I'm actually going to walk this back to 1.17 to match the rest of the codebase. We should move this forward together.